src/RewardsStreamerMP.sol

```diff
@@ -316,29 +411,32 @@ contract RewardsStreamerMP is
         lastRewardTime = block.timestamp < rewardEndTime ? block.timestamp : rewardEndTime;
     }
 
-    function _calculateBonusMP(uint256 amount, uint256 lockPeriod) internal view returns (uint256) {
+    function _calculateBonusMP(uint256 amount, uint256 lockPeriod) internal pure returns (uint256) {
         uint256 lockMultiplier = (lockPeriod * MAX_MULTIPLIER * SCALE_FACTOR) / MAX_LOCKUP_PERIOD;
         return amount * lockMultiplier / SCALE_FACTOR;
     }
 
-    function _updateAccountMP(address accountAddress) internal {
-        Account storage account = accounts[accountAddress];
-
+    function _getAccruedMP(Account storage account) internal view returns (uint256) {
```

Comment:
very small thing, shall we call this `_getAccountAccruedMP` so that it's clear it's for the account? just in case we extract the global MP logic to something with the same name

Reply:
I agree with renaming this.

I think we should do this across the board for all other things as well https://github.com/vacp2p/staking-reward-streamer/issues/87